v2/api/insights/v1api20200202/structure.txt

```diff
@@ -22,6 +22,10 @@ github.com/Azure/azure-service-operator/v2/api/insights/v1api20200202
 │   │   │   └── "LogAnalytics"
 │   │   ├── Kind: *string
 │   │   ├── Location: *string
+│   │   ├── OperatorSpec: *Object (1 property)
+│   │   │   └── ConfigMaps: *Object (2 properties)
+│   │   │       ├── ConnectionString: *genruntime.ConfigMapDestination
```

Comment:
Aren't these secrets, not ConfigMaps?

Shouldn't they be going to secrets?

Or do they not actually contain any secret values?

Reply:
I thought the same as well and investigated a bit. Azure docs state its not a secret [here](https://learn.microsoft.com/en-us/azure/azure-monitor/app/sdk-connection-string?tabs=net#is-the-connection-string-a-secret)